boards/samr21-xpro/board.c

```diff
@@ -50,3 +50,8 @@ void board_init(void)
     /* initialize the CPU */
     cpu_init();
 }
+
+void __attribute__((weak)) board_init(void)
+{
+    board_samr21_xpro_init();
+}
```

Comment:
Just out of curiosity, would this also work:

```C
void __attribute__((weak,alias("board_samr21_xpro_init"))) board_init(void);
```

This would have no ROM overhead and should still allow overwriting `board_init()`.

Reply:
Apparently, I had the same idea before: https://github.com/RIOT-OS/RIOT/pull/16481#issuecomment-843954269